components/ScheduleSection.js

```diff
@@ -0,0 +1,240 @@
+          />
+        </SessionRow>
+      </div>
+      <style jsx>{`
+        .schedule {
+          text-align: center;
+        }
+        .container {
+          max-width: 1060px;
+          width: 100%;
+          margin: 0 auto;
+          text-align: left;
+        }
+        @media (max-width: 768px){
+          .container {
+            max-width: 750px;
+            text-align: center;
+          }
+        }
+      `}</style>
+    </div>
+  )
+}
+
+const getSpeakerAvatar = (name) => {
+  return `static/speakers/${name.replace(' ', '')}.jpg`
+}
+
+const ScheduleRow = ({ children }) => (
+  <div className='row _margin-bottom-30'>
+    {children}
+    <style jsx>{`
+      .row {
+        position: relative;
+        display: flex;
+        flex-wrap: wrap;
+        padding-left:7.5px;
+        padding-right:7.5px;
+      }
+      ._margin-bottom-30 {
+        margin-bottom: 30px;
+      }
+    `}</style>
+  </div>
+)
+
+const ScheduleColumn = ({ children, size }) => {
+  return (
+    <div className='column' data-size={size}>
+      {children}
+      <style jsx>{`
+        .column { width: 100%; }
+        @media (min-width: 768px) {
+          .column[data-size="2"] { width: 16.66%; }
+          .column[data-size="5"] { width: 41.65%; }
+          .column[data-size="8"] { width: 66.64%; }
+          .column[data-size="10"] { width: 83.33%; }
+        }
+      `}</style>
+    </div>
+  )
+}
+
+const ActivityRow = ({ time, children }) => (
+  <ScheduleRow>
+    <ScheduleColumn size={2}>
+      <SessionItemTime>{time}</SessionItemTime>
+    </ScheduleColumn>
+    <ScheduleColumn size={10}>
+      <div className='main'>{children}</div>
+    </ScheduleColumn>
+    <style jsx>{`
+      .main {
+        font-size:28px;
+      }
+    `}</style>
+  </ScheduleRow>
+)
+
+const BreakRow = ({ time }) => (
+  <ScheduleRow>
+    <ScheduleColumn size={2}>
+      <SessionItemTime>{time}</SessionItemTime>
+    </ScheduleColumn>
+    <ScheduleColumn size={8}>
+      <div className='container-break'><div className='break' /></div>
+    </ScheduleColumn>
+    <style jsx>{`
+      .break {
+        height: 2px;
+        width: 20%;
+        background-color: #3A3A3A;
+      }
+      .container-break {
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+      @media (max-width: 768px){
+        .container-break {
+          margin-top: 15px;
+        }
+      }
+    `}</style>
+  </ScheduleRow>
+)
+
+const SessionRow = ({ time, children }) => (
+  <ScheduleRow>
+    <ScheduleColumn size={2}>
+      <SessionItemTime>{time}</SessionItemTime>
+    </ScheduleColumn>
+    {children}
+  </ScheduleRow>
+)
+
+const SessionItemTime = ({ children }) => (
+  <div className='time'>
+    {children}
+    <style jsx>{`
+      .time {
+        font-size:28px;
+      }
+    `}</style>
+  </div>
+)
+
+const SessionItem = ({ topic, speakerName, speakerAvatar, speakerTitle }) => (
+  <ScheduleColumn size={5}>
```

Comment:
Here as well, speaker’s avatar is not vertically aligned when height differ. This can be fixed by using `<ScheduleColumn size={5} alignSelf='center'>`

![image](https://cloud.githubusercontent.com/assets/193136/26300442/4baa07f0-3f08-11e7-908f-1f6c71089771.png)